Makefile

```diff
@@ -9,11 +9,11 @@ MODULE := $(shell head -1 go.mod | cut -d' ' -f2)
 
 ARTIFACTS := _out
 TEST_PKGS ?= ./...
-TALOS_RELEASE ?= v1.4.0
-DEFAULT_K8S_VERSION ?= v1.26.0
+TALOS_RELEASE ?= v1.5.0
+DEFAULT_K8S_VERSION ?= v1.27.2
```

Comment:
isn't 1.27.4 the latest?

Reply:
I used same as base CAPI version in this release. I think we could use almost anything, as it's just in the tests only